bin/refresh-verify-change.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#/ Builds and verifies the specified plan and any reverse dependencies
```

Comment:
Does this script exit on the first build error or test failure?
If so it might be nice to modify it (later, not in this PR) to always try to build and test everything and report a single aggregate result.

Reply:
Yes, and I think it has to. If we were to not fail on a build error plans after that in the build order would potentially not be built against the updated packages.